networking/managing-networking.adoc

```diff
@@ -0,0 +1,28 @@
+// Assembly filename: managing-networking.adoc
+// Explains network management
+[id='managing-networking’]
```

Comment:
@ariordan-redhat, it looks like you have a backtick instead of a single quotation mark in this id. [id='managing-networking'] might fix your build.